dbld/builddeps

```diff
@@ -81,8 +81,41 @@ function install_gosu() {
     chmod +x /usr/local/bin/gosu
 }
 
+# NOTE: libjvm.so is not anymore in the java lib dir itself, a better choice could be to use libnio.so to find the real lib dir
+#       for backward compatibility the libjvm.so is kept as the default search base
+function get_jvm_paths() {
+    LIB_FILE=libjvm.so
+    if [ $# -gt 0 ]; then
+        LIB_FILE=$1
+    fi
+    find / -name "${LIB_FILE}" 2>/dev/null | sed "s@/${LIB_FILE}@@g"
+}
+
+function set_jvm_envs() {
```

Comment:
this function is introduced here, but I don't see a user. where is this used from?

Reply:
This is the main function that solves the internal build issues, as the environment manipulation partially happens in these scripts I just added the needed extensions next to the already existing ones

Reply:
I am not against adding this per-se. But it would be a lot better to have visibility into what uses this. Do you think you could publish the docker file that uses "dbld/builddeps"? or you are using it in a different context?

I am sorry but I don't really have a recollection of the internal tools, but adding this function here, without seeing why this is needed would be a fragile situation.

At least some comments on top would be useful here.